WPMedia/fusion-news-theme-blocks
# Added dir attribute to html tag

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

This PR should be merged in AFTER https://github.com/WPMedia/arc-themes-feature-pack/pull/434 , which updates the feature pack blocks.json
## Jira Ticket

- [THEMES-1181](https://arcpublishing.atlassian.net/browse/THEMES-1181)

## Acceptance Criteria

The text direction value can map directly to the dir attribute on the <html> tag in our default output type block.

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout branch THEMES-1181 in both the blocks and feature pack repos `git checkout 1181`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block`
3. Open a local page, such as: http://localhost/pf/2022/08/11/on-oregons-central-coast-a-low-key-exploration-of-wineries-restaurants-and-nature/?_website=the-gazette
4. Open the dev tools and navigate to the elements tab
5. Verify that the HTML tag pulls in the textDirection value from the feature pack blocks.json

## Effect Of Changes

### Before

HTML tag did not include a "dir" attribute
<img width="215" alt="image" src="https://github.com/WPMedia/arc-themes-blocks/assets/85515364/19f18f29-f199-42a2-ad2a-b7dfc9125cf7">


### After

HTML tag includes the "dir" attribute which pulls in the value set in the feature pack blocks.json
<img width="245" alt="image" src="https://github.com/WPMedia/arc-themes-blocks/assets/85515364/e89754f5-93f3-43b5-8861-466d7f65419e">


## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1181]: https://arcpublishing.atlassian.net/browse/THEMES-1181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
